src/App.js

```diff
@@ -1,25 +1,14 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+import Calculator from './components/Calculator';
 
-export default App;
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
In this case, adding a constructor to your class-based component should help you fix the issue, it may look something like this :point_down: 

```React 
constructor() {
    super();
    this.state = {};
  }